src/pages/Report.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Outlet } from "react-router-dom";
+import { Container } from "../components/styles/Container.styled";
+import Header from "../components/Header";
+import mailIcon from "../assets/imgs/mail.svg";
+import importantIcon from "../assets/imgs/important_vector.png";
+import { SimpleFooter } from "../components/SimpleFooter";
+
+const Report = () => {
+  const [message, setMessage] = useState("");
+
+  function handleMessage(e: any): void {
+    setMessage(() => e.target.value);
+  }
+  
+  const MAX_LENGTH = 2000;
+  return (
+    <div>
+      <Header title="Central de denúncias" backTo="component"/>
```

Comment:
Deixa esse backTo sem rota, pois ainda precisamos alinhar isso com a equipe de UI/UX
```suggestion
      <Header title="Central de denúncias" backTo=""/>
```

Reply:
@CarlosJunioor atenção a esses comentários.

Reply:
ta feiot

Reply:
Good job!